fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -2271,6 +2271,11 @@ public void setDetailShapePlanNodes(String detailShapePlanNodes) {
                 "use other health replica when the use_fix_replica meet error" })
     public boolean fallbackOtherReplicaWhenFixedCorrupt = false;
 
+    @VariableMgr.VarAttr(name = "FE_DEBUG", needForward = true, fuzzy = true,
```

Comment:
```suggestion
    public static final String DEBUG_NEREIDS_THROW_ALL_EXCEPTIONS = "debug_nereids_throw_all_exceptions";


    @VariableMgr.VarAttr(name = DEBUG_NEREIDS_THROW_ALL_EXCEPTIONS, needForward = true, fuzzy = true,
```

Reply:
not only for nereids, I think this var can be used by any part of FE.